playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -0,0 +1,111 @@
+@import "../tokens/positioning";
+@import "../tokens/colors";
+@import "../pb_card/card_mixin";
+@import "../tokens/shadows";
+
+@keyframes modalFadeIn {
+	from {
+		transform: translate3d(0, -100%, 0);
+		opacity: 0;
+	}
+	to {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+}
+
+@keyframes modalFadeOut {
+	from {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+	to {
+		transform: translate3d(0, -50%, 0);
+		opacity: 0;
+	}
+}
+
+@keyframes overlayFade {
+	from {
+		opacity: 0;
+		transform: translateY(0);
+	}
+	to {
+		opacity: 1;
+		transform: translateY(0);
+	}
+}
+
+@keyframes overlayFadeOut {
+	from {
+		opacity: 1;
+	}
+	to {
+		opacity: 0;
+	}
+}
+
+.dialog {
```

Comment:
This doesn't seem to be following the `pb_` class pattern. Can you align?

Reply:
I can align this one, the dialog classname to be pb_dialog... but certain ones have to stay the same because they are triggers for the `react-modal` library.

Reply:
Got it working actually. Scratch all that  ^^^^.